server/app/api.py

```diff
@@ -2108,31 +2108,48 @@ class FinalSubmissionAPI(APIResource):
     The API resource for the Assignment Object
     """
     model = models.FinalSubmission
-
+    contains_entities = False
+    
     methods = {
         'get': {
         },
         'index': {
         },
-        'post': {
+        'mark_backup': {
+            'methods': set(['POST']),
             'web_args': {
-                'submission': KeyArg('Submission', required=True)
```

Comment:
For some reason, the front end 'submission' object was actually a backup. You have a point though. I'll restore the `post` method but leave this in.